package.json

```diff
@@ -72,9 +73,11 @@
     "lint": "eslint src/**",
     "plaintest": "mocha --check-leaks -b -R spec test/index.js && npm run tape",
     "prepublish": "npm run babel",
+    "docker_test": "bash ./scripts/docker-for-test.js",
```

Comment:
I just noticed that this shell script is actually .js, should be .sh I'll try this thing on windows and check if it works correctly